### PR TITLE
Reports service to abstract APIs

### PR DESCRIPTION
**Pull request by constanzauanini**
Add a report service with all the apis
Files and photos management may be reviewed when using it

### CODE REVIEW

File: src/services/ReportService.ts [hunk 1]
```diff
@@ -0,0 +1,361 @@
...
+      '{reportId}': reportId.toString(),
+      '{fileId}': fileId.toString(),
+    },
+  });
+};
+
+/**
+ * Exported functions
+ */
+const ReportService = {
```
Comment (karthikbtf) [inline src/services/ReportService.ts:342]: lgtm.. would it be worth splitting this into ReportService, ReportLockingService and ReportPhotosService? or is that overkill.. i guess my thoughts are mostly coming from size of service (maybe that's not a valid issue).. cc @charlie-tf @constanzauanini

Comment (karthikbtf) [reply src/services/ReportService.ts:342]: not requesting any changes.. was mostly a question to hear y'alls thoughts

Comment (constanzauanini) [reply src/services/ReportService.ts:342]: Yes, I don't think it's completely necessary, but if we prefer small files I can split it

Comment (charlie-tf) [reply src/services/ReportService.ts:342]: I'm kinda on the side of keeping it all in one. Then there's just one service to interact with. But either way would completely work 🤷

Comment (karthikbtf) [reply src/services/ReportService.ts:342]: Thanks Constanza/Charlie.. one files sounds good 👍

**Review by charlie-tf — APPROVED** [review]
lgtm!

**Pull request merged**